nc_py_api/options.py

```diff
@@ -33,22 +33,13 @@
 SSL certificates (a.k.a CA bundle) used to  verify the identity of requested hosts. Either **True** (default CA bundle),
 a path to an SSL certificate file, or **False** (which will disable verification)."""
 str_val = environ.get("NPA_NC_CERT", "True")
-# https://github.com/encode/httpx/issues/302
-# when "httpx" will switch to use "truststore" by default - uncomment next line
```

Comment:
And there it is! Niquests have a built in way to access the OS trust store! Without injection/monkeypatching!